src/components/Activities/Activity2/Game/FillInBlankGamePage.tsx

```diff
@@ -126,12 +115,6 @@ function FillInBlankGamePage(props: FillInBlankGamePageProps): JSX.Element {
         </div>
       </div>
       <div className='scene-info'>
-        <div className='icon'>
-          <img src={ClockSvg} />
-          <span className='details'>
-            {((currTime - startTime) / 1000).toFixed(2)} seconds
-          </span>
-        </div>
         <div className='icon'>
           <img src={SceneSvg} />
           <span className='details'>
```

Comment:
oops shoulda caught this earlier, but you have a css `.scene-info .details` but you don't use it because the details is under `.scene-info .icon .details`
```suggestion
        </div>
        <span className='details'>
```
probs need to delete a `</div>` below as well

Reply:
ok I did this. I didn't accept your suggestion bcuz I wanted to do all the necessary changes in one commit